02-express-tutorial/02-http-app.js

```diff
@@ -0,0 +1,52 @@
+const http = require('http');
+const { readFileSync } = require('fs');
+
+//get all files
+const homePage = readFileSync('./navbar-app/index.html')
+const homeStyles = readFileSync('./navbar-app/styles.css')
+const homeImage = readFileSync('./navbar-app/logo.svg')
+const homeLogic = readFileSync('./navbar-app/browser-app.js')
+
+const server = http.createServer((req, res) => {
+    //console.log(req.method)
+
+    const url = req.url;
+    //home page
+    if (url === '/'){
```

Comment:
nit: always put a space between the closing parentheses and opening curly brace for if statements, for loops, and function definitions

```javascript
if (condition) {
  // ...
}

function foo() {
  // ...
}

for (let i = 0; i < 100; i++) {
  // ...
}

for (foo of bar) {
 // ...
}
```